doc/classes/GPUParticles2D.xml

```diff
@@ -50,6 +50,10 @@
 		<member name="amount" type="int" setter="set_amount" getter="get_amount" default="8">
 			Number of particles emitted in one emission cycle.
 		</member>
+		<member name="amount_ratio" type="float" setter="set_amount_ratio" getter="get_amount_ratio" default="1.0">
+			The ratio of particles that should actually be emitted. If set to a value lower than [code]1.0[/code], this will set the amount of emitted particles throughout the lifetime to [code]amount * amount_ratio[/code]. Unlike changing [member GPUParticles2D.amount], changing [member amount_ratio] while emitting does not affect already-emitted particles and doesn't cause the particle system to restart. [member amount_ratio] can be used to create effects that make the number of emitted particles vary over time.
```

Comment:
```suggestion
			The ratio of particles that will actually be emitted. If set to a value lower than [code]1.0[/code], the amount of emitted particles throughout the lifetime will be [code]amount * amount_ratio[/code]. Unlike the [member amount], this property can be changed without affecting already-emitted particles or causing the particle system to restart. [member amount_ratio] can be used for effects that vary the number of emitted particles over time.
```

It was really long, let's try this instead? Same for 3D.

The line below still uses `[member GPUParticles2D.amount]` instead of just `[member amount]`.